packages/fether-react/package.json

```diff
@@ -35,10 +35,10 @@
   },
   "dependencies": {
     "@craco/craco": "^3.2.3",
-    "@parity/api": "^2.1.22",
-    "@parity/contracts": "^1.0.0",
-    "@parity/light.js": "^1.0.9",
-    "@parity/light.js-react": "^1.0.0",
+    "@parity/api": "^3.0.10",
+    "@parity/contracts": "^3.0.10",
+    "@parity/light.js": "^3.0.10",
+    "@parity/light.js-react": "^3.0.10",
```

Comment:
@parity/light.js-react is at 3.0.11 https://www.npmjs.com/package/@parity/light.js-react